<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModuleContourWidget</class>
 <widget class="QWidget" name="ModuleContourWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>246</width>
    <height>192</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="cbColorMapData">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Color Map Data</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="cbSelectColor">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Select Color</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="pqColorChooserButton" name="colorChooser" native="true"/>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QCheckBox" name="cbColorByArray">
       <property name="text">
        <string>Color by Array</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboColorByArray">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Value</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="tomviz::DoubleSliderWidget" name="sliValue" native="true"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Mode</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="cbRepresentation"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Opacity</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="tomviz::DoubleSliderWidget" name="sliOpacity" native="true"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>tomviz::DoubleSliderWidget</class>
   <extends>QWidget</extends>
   <header>DoubleSliderWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>pqColorChooserButton</class>
   <extends>QWidget</extends>
   <header>pqColorChooserButton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>cbColorByArray</sender>
   <signal>toggled(bool)</signal>
   <receiver>comboColorByArray</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>187</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbColorByArray</sender>
   <signal>toggled(bool)</signal>
   <receiver>cbSelectColor</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>63</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbColorByArray</sender>
   <signal>toggled(bool)</signal>
   <receiver>colorChooser</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>122</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
